great_expectations/core/usage_statistics/schemas.py

```diff
@@ -1028,6 +1028,7 @@
                         "data_context.open_data_docs",
                         "data_context.run_checkpoint",
                         "expectation_suite.add_expectation",
+                        "data_context.run_profiler_with_dynamic_arguments",
                     ],
                 },
                 "event_payload": {"$ref": "#/definitions/empty_payload"},
```

Comment:
Correct, this is empty since it will eventually call the `profiler.run()` method and emit a `profiler.run` event with a payload containing information about how the profiler was run.